docs/Qualitaetssicherung.tex

```diff
@@ -68,6 +68,29 @@ \section{Änderungen in der Implementierung}
 \end{itemize}
 
 \section{Unittests}
+Die Phase der Qualitätssicherung haben wir damit begonnen, die Unittests aus der Implementierungsphase weiter auszubauen und so die Testabdeckung zu maximieren.
+Innerhalb des Models war die Testabdeckung bereits sehr gut. Für die Controller und die graphischen Benutzeroberflächen waren noch keine Unittests vorhanden.
+
+Für die Controller blieben wir bei der Entscheidung, keine Unittests einzuführen, da diese hier kaum anwendbar waren. Die Entscheidung aus der Implementierungsphase,
+für die Controller nur Integrationstests zu verwenden wurde beibehalten. Für die beiden graphischen Benutzeroberflächen haben wir jedoch noch Testklassen
+erstellt. Hier konnte nur eine geringere Abdeckung erreicht werden als es im Model der Fall ist, dennoch konnte in beiden Oberflächen eine Zeilenabdeckung
```

Comment:
werden [Komma] als es im Model der Fall ist, dennoch [liegt] in beiden Oberflächen [die] Zeilenabdeckung [bei] über 50\%. (Hier doppelt sich das erreicht werden in einem zu nahem und engen Kontext)